tools/AzureRM/AzureRM.psd1

```diff
@@ -51,7 +51,8 @@ CLRVersion = '4.0'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '5.0.0'; }, 
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.0.0'; }, 
+               @{ModuleName = 'Azure.Storage'; ModuleVersion = '4.3.0'; }, 
```

Comment:
Should this be 4.4.0?

Reply:
No, you cnanot take a dependency on a preview module (this is a >= dependency for that reason, however.